pandas/tseries/tests/test_timedeltas.py

```diff
@@ -944,6 +945,27 @@ def test_fields(self):
 
         tm.assert_series_equal(s.dt.days,Series([1,np.nan],index=[0,1]))
         tm.assert_series_equal(s.dt.seconds,Series([10*3600+11*60+12,np.nan],index=[0,1]))
+        
+    def test_total_seconds(self):
+        # test index
+        rng = timedelta_range('1 days, 10:11:12.100123456', periods=2, freq='s')
+        expt = [1*86400+10*3600+11*60+12+100123456./1e9,1*86400+10*3600+11*60+13+100123456./1e9]
+        assert_allclose(rng.total_seconds, expt, atol=1e-10, rtol=0)
+        
+        # test Series
+        s = Series(rng)
+        s_expt = Series(expt,index=[0,1])
+        tm.assert_series_equal(s.dt.total_seconds,s_expt)
```

Comment:
can you add a test for the scalar as well (even though we know it works as its a sub-class of ``timedelta``), I don't think its testsed. e.g. ``Timedelta(....).total_seconds()``.

Note that this is a method, so maybe we should make ``total_seconds`` (on a ``TimedeltaIndex``) a method as well (even though its a bit odd, for consitency with the scalar)?